android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright @ 2019-present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.meet.sdk;
+
+import android.app.Activity;
+import android.app.Notification;
+import android.app.Service;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.os.IBinder;
+import android.util.Log;
+
+import java.util.Random;
+
+public class JitsiMeetOngoingConferenceService extends Service implements OngoingConferenceTracker.OngoingConferenceListener {
+    private static final String TAG = JitsiMeetOngoingConferenceService.class.getSimpleName();
+
+    private static final int NOTIFICATION_ID = new Random().nextInt(99999) + 10000;
+
+    static final class Actions {
+        static final String ONGOING_CONFERENCE = TAG + ":ONGOING_CONFERENCE";
+        static final String HANGUP = TAG + ":HANGUP";
+    }
+
+    private static Class pendingIntentActivityClass;
```

Comment:
Don't really like this here. It will still persist information after you call stop and this is destroyed
<br> You seem to have access to the activity class through React context or you can pass it as a parameter on the intent since Class is serializable

Reply:
How about I get rid of it now and we add it if and when we need it?